spec/middleman_prismic_spec.rb

```diff
@@ -8,7 +8,22 @@
         FakePrismic.set_document(id: "referoo", type: "demo-test")
         change_directory_and_configure_with_prismic(directory, server)
 
-        run_middleman_prismic
+        `middleman prismic`
+
+        hex_string = Digest::MD5.hexdigest("referoo")
+        document = YAML.load_file("data/prismic/demo-tests/#{hex_string}.yml")
+
+        expect(document.id).to eq "referoo"
+      end
+    end
+
+    it "download content from prismic by reference"  do |example|
+      Capybara::Discoball.spin(FakePrismic) do |server|
```

Comment:
should this move up into a the `around`? so, instead of `:in_temp_directory`, maybe a `:fake_prismic`. that would let you clean up `directory = example.metadata[:directory]`

Reply:
That's a really good point!

Reply:
One concern I have about moving this into a surround like that is that I would need to move the `config.rb` creation into that around as well. I'm not as thrilled about that. Also, it seems I don't need to have the `directory` after all. I removed the thing that was using it.